app/entry.server.tsx

```diff
@@ -55,7 +54,7 @@ function handleBotRequest(
           responseHeaders.set("Content-Type", "text/html");
 
           resolve(
-            new Response(body, {
+            new Response(Readable.toWeb(body) as ReadableStream, {
```

Comment:
We should switch these to use our internal utility (which I was previously unaware of): https://github.com/remix-run/remix/pull/7361